backend/models/userModel.js

```diff
@@ -0,0 +1,14 @@
+const connection = require('../database');
+
+const checkEmailExists = (email, callback) => {
```

Comment:
Just for safety's sake, I would use Prepared statements:
https://sidorares.github.io/node-mysql2/docs#using-prepared-statements

Essentially, you need to change the functions to use `connection.execute` instead of `connection.query`